core/cmd/client_test.go

```diff
@@ -305,3 +310,17 @@ func TestFileSessionRequestBuilder(t *testing.T) {
 		})
 	}
 }
+
+func TestNewUserCache(t *testing.T) {
+
+	r, err := rand.Int(rand.Reader, big.NewInt(256*1024*1024))
+	subDir := filepath.Base(fmt.Sprintf("%s-%d", t.Name(), r.Int64()))
```

Comment:
Would it make sense to use `t.TempDir()`? https://pkg.go.dev/testing#T.TempDir

Reply:
no, t.TempDir and ioutil.TempDir don't fit well here. Added a comment.